### PR TITLE
feat: Android share extension

### PR DESCRIPTION
**Pull request by alexandra-marin**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/5957/android-build-share-with-peerio-extension
#### Testing instructions  
Share external image/video/pdf files to Peerio Android.
Should be no regressions for iOS share.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: android/app/src/main/java/com/peerio/app/MainActivity.java [hunk 1]
```diff
@@ -58,4 +60,57 @@ public void invokeDefaultOnBackPressed() {
     protected String getMainComponentName() {
         return "peeriomobile";
     }
+
+    public static class TestActivityDelegate extends ReactActivityDelegate {
```
Comment (seavan) [inline android/app/src/main/java/com/peerio/app/MainActivity.java]: @alexandra-marin why is the class static? is it used anywhere? I cannot find any reference in this PR.

I've thought it's going to be handling this events like this:
https://developer.android.com/training/sharing/receive

In the main activity you would add the handling as described.

Comment (seavan) [reply android/app/src/main/java/com/peerio/app/MainActivity.java]: ok I see. please:

- move TestActivityDelegate to a separate Java file
- remove `static` class keyword, you are getting `this` activity as a parameter and there's no reason to keep this class inside MainActivity as far as I can see.
- rename TestActivityDelegate to something more clear, maybe LaunchActivityDelegate.

thanks.

**Comment by alexandra-marin** [comment]
@seavan ready for review

**Comment by ebilg** [comment]
can be merged

**Pull request merged**